workflows/basic.yml

```diff
@@ -0,0 +1,39 @@
+name: Coverage
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
Should this be `master` instead of `main`, given we don't have a branch called "main" in the repo?